src/SimpleExcelWriter.php

```diff
@@ -87,6 +87,13 @@ public function addRow($row, Style $style = null)
         return $this;
     }
 
+    public function addRows(array $rows)
```

Comment:
Might be nice to accept an `iterable` here instead of just an array

Reply:
Done, let's hope all the tests pass!